public/javascripts/SVLabel/src/SVLabel/mission/CVGroundTruthMission.js

```diff
@@ -0,0 +1,173 @@
+function CVGroundTruthMission(mission) {
+    var self = this;
+
+    /**
+     * If an existing ground truth CV audit mission is available, initialize the mission. Otherwise, display
+     * form to create a new CV ground truth audit mission.
+     */
+    function _init(mission) {
+        self.verifyAndSubmitCVGroundTruthAuditPanoids = verifyAndSubmitCVGroundTruthAuditPanoids;
```

Comment:
I think in the rest of the audit front-end these lines: `self.funcName = funcName;` are all at the end of the file before the `return this;`. It's cool if that doesn't apply for your code that's fine, just wanted to make sure we aren't accidentally doing something different in this file.